tests/test_dimension.py

```diff
@@ -170,6 +170,57 @@ def test_basic(self):
         assert np.all([np.allclose(usave.data[i], i*factor)
                       for i in range((nt+factor-1)//factor)])
 
+    def test_spacial_subsampling(self):
+        """
+        Test conditional dimension for the spatial ones.
+        This test saves u every two grid points :
+        u2[x, y] = u[2*x, 2*y]
+        """
+        nt = 19
+        grid = Grid(shape=(12, 12))
+        time = grid.time_dim
+
+        u = TimeFunction(name='u', grid=grid, save=nt)
+        assert(grid.time_dim in u.indices)
+
+        # Creates subsampled spatial dimensions and accordine grid
+        dims = tuple([ConditionalDimension(d.name+'sub', parent=d, factor=2)
+                      for d in u.grid.dimensions])
+        grid2 = Grid((6, 6), dimensions=dims)
+        u2 = TimeFunction(name='u2', grid=grid2, save=nt)
+        assert(time in u2.indices)
+
+        eqns = [Eq(u.forward, u + 1.), Eq(u2, u)]
+        op = Operator(eqns)
+        op.apply(time_M=nt-2)
+        # Verify that u2[x,y]= u[2*x, 2*y]
+        assert np.allclose(u.data[:-1, 0:-1:2, 0:-1:2], u2.data[:-1, :, :])
+
+    def test_subsampled_fd(self):
```

Comment:
this test should probably go to ``test_finite_differences.py`` or ``test_derivatives.py``, I don't remember

Reply:
Not sure, this one is a weird test can can go with FD/dimension so not sure where it is best to put.

Reply:
OK, I think we should split this test into two tests. The first one should be about the symbolic interface only, and could go to one of the two files above (btw, should we merge them?)

Reply:
>btw, should we merge them?

Probably yes